src/Engine/labyrinthFactory.h

```diff
@@ -0,0 +1,25 @@
+//
+// Created by nikit on 15/04/2018.
+//
+
+#ifndef ALGODATII_LABYRINTHFACTORY_H
+#define ALGODATII_LABYRINTHFACTORY_H
+
+typedef struct labyrinth {
+    const int firstDimensionSize;
+    const int secondDimensionSize;
+    const int freeField = 0;
+    const int filledField = 1;
+    int array [firstDimensionSize][secondDimensionSize];
+};
+
+class labyrinthFactory {
```

Comment:
Brauchen wir wirklich ne Factory Klasse?

Reply:
ja, ich wollte eigentlich wissen, was deine Gründe sind/waren, eine Factory Klasse einzufügen. :) 
Weil ich kenn Factory Klassen nur in diesem Zusammenhang https://de.wikipedia.org/wiki/Abstrakte_Fabrik